modules/core/src/lib/loader-utils/get-fetch-function.ts

```diff
@@ -25,7 +25,7 @@ export function getFetchFunction(
 
   // options.fetch can be an options object
   if (isObject(fetchOptions.fetch)) {
-    return (url) => fetchFile(url, fetchOptions as RequestInit);
```

Comment:
It doesn't really make sense to call this object `fetchOptions` - it is a `loaderOptions` object.